### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BirmacherAkos**
Deprecation note.

### CODE REVIEW

**Review by lszucs — COMMENTED** [review]
LGTM

**Review by lpusok — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
+WARNING: This step is deprecated, use the AVD Manager step instead:
```
Comment (trapacska) [inline README.md]: It is rather a feedback but now it looks like:
screen shot 2019-02-25 at 10 35 12 am
And in one of our already deprecated steps it looks like: 
screen shot 2019-02-25 at 10 35 05 am

Comment (trapacska) [reply README.md]: The removal date is missing.

**Review by trapacska — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BirmacherAkos) [reply README.md]: Fixed in:
https://github.com/bitrise-steplib/steps-create-android-emulator/pull/27/commits/bc65b77f432cc2c9b5a009ce7f78c59bbf51bc32

**Review by trapacska — APPROVED** [review]

**Review by godrei — APPROVED** [review]

**Pull request merged**